test/test_event.py

```diff
@@ -36,6 +38,44 @@ def test_event_id_recomputes(self):
         assert event.id != event_id
     
 
+    def test_note_id_bech32_conversion(self):
+        """ should convert the event id to its `note`-prepended bech32 form """
+        event = Event(content="some event")
+        assert event.note_id.startswith("note")
+
+        # reverse the bech32 encoding
+        hrp, data, spec = bech32.bech32_decode(event.note_id)
+        raw_event_id = bech32.convertbits(data, 5, 8)[:-1]
+
+        # Should get the same event_id back
+        assert event.id == hexlify(bytes(raw_event_id)).decode()
```

Comment:
Instead of `hexlify` can we do `bytes(raw_event_id).hex()` to avoid the extra import?